getting_started/workflow/export/exporting_pcks.rst

```diff
@@ -0,0 +1,129 @@
+.. _doc_exporting_pcks:
+
+Exporting packs, patches, and mods
+==================================
+
+Use cases
+---------
+
+Oftentimes one would like to add functionality to one's game after it has been
+deployed.
+
+Examples of this include...
+
+- Downloadable Content: the ability to add features and content to one's game.
+- Patches: the ability to fix a bug that is present in a shipped product.
+- Mods: grant other people the ability to create content for one's game.
+
+These tools help developers to extend their development beyond the initial
+release.
+
+Overview of \*.pck files
+------------------------
+
+Godot enables this via a feature called **resource packs**
+(\*.pck files).
+
+**Advantages:**
+
+- incremental updates/patches
+- offer DLCs
+- offer mod support
+- no source code disclosure needed for mods
+- more modular project structure
+- users don’t have to replace the entire game
+
+The first part of using them involves exporting and delivering the project to
+players. Then, when one wants to add functionality or content later on, they
+just deliver the updates via \*.pck file to the users.
+
+\*.pck files usually contain, but are not limited to:
+
+- scripts
+- scenes
+- shaders
+- models
+- textures
+- sound effects
+- music
+- any other asset suitable for import into the game
+
+The \*.pck files can even be an entirely different Godot project, which the
+original game loads in at runtime.
+
+Generating \*.pck files
+-----------------------
+
+In order to pack all resources of a project into a \*.pck file open the project
+and go to Project/Export and click on “Export PCK/Zip”. Also make sure to have
+an export template selected while doing so.
+
+.. image:: img/export_pck.png
+
+Another method would be to `export from the command line
+<https://docs.godotengine.org/en/latest/getting_started/editor/command_line_tutorial.html#exporting>`_.
```

Comment:
This should use `:ref:` instead of a hardcoded link I think. However, to link directly to the "Exporting" section a new reST label in `getting_started/editor/command_line_tutorial.rst` would be needed because apparently you can't use anchors with `:ref:`. :(

Maybe we could add the [Sphinx autosection extension](http://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html) to automate this.

Reply:
I just know that other sections written by more experienced contributors had done it that way, but if we can get a more official, Sphinx-like solution in place, that'd be preferred, sure.

Reply:
Then more experienced contributors were wrong :)

Internal hyperlinks should indeed use `:ref:`, as do the vast majority of them:
```
$ rg ":ref:" | wc -l
16793
$ rg "http[s]?://docs.godotengine.org/en/" | wc -l
3
```
Actually there's a single occurrence of such a direct link (the other two are just in the layout page and cross-reference between master and stable docs):
```
getting_started/scripting/gdscript/static_typing.rst
107:   types <http://docs.godotengine.org/en/3.0/getting_started/scripting/gdscript/gdscript_basics.html#built-in-types>`__
```

Reply:
You can reference the whole page `:ref:doc_command_line_tutorial`. To reference the "Exporting" section directly, you'd have to add a new label in that page indeed, with:
```
.. _doc_command_line_tutorial_exporting:
```
(before the "Exporting" title)
which you can then reference with `:ref:doc_command_line_tutorial_exporting`.

Reply:
I'll merge and fix those two occurrences myself :)

Reply:
To be fair there's two other ones using another URL:
```
$ rg "http[s]?://godot.readthedocs.io/en/"
getting_started/step_by_step/resources.rst
225:If the script's language supports `script classes <https://godot.readthedocs.io/en/latest/getting_started/step_by_step/scripting_continued.html#register-scripts-as-classes>`__,

getting_started/scripting/gdscript/static_typing.rst
111:   `class_name <https://godot.readthedocs.io/en/latest/getting_started/step_by_step/scripting_continued.html#register-scripts-as-classes>`__
```

Reply:
FYI: 8a49da6d